docs/AboutUs.md

```diff
@@ -40,7 +40,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Ryan Walter Lee
 
-<img src="images/ryanwalterlee.png" width="200px">
+<img src="images/RyanWalterLee.png" width="200px">
```

Comment:
change both your file name and the name here to "ryanwalterlee.png", supposed to be your github username

Reply:
changed!